### ISSUE TITLE
bootstrap 4 support

### ISSUE THREAD

**Issue by logicminds** [creativetimofficial/ct-light-bootstrap-dashboard-pro-react#13]
Is this compatible with bootstrap 4?

**Comment by einazare** [comment]
Hello @logicminds 

And thank you for your interest in our product.
This product uses React-Bootstrap which as of moment is based on bootstrap 3.
You can find out more abour React-Bootstrap here: [https://react-bootstrap.github.io/](https://react-bootstrap.github.io/).

Best,
Manu

**Comment by fellars** [comment]
Any thoughts on what it would take to switch out React-Bootstrap for this project?
https://reactstrap.github.io/

in order to get BS4 support

**Comment by einazare** [comment]
Hello @fellars ,

Thank you for your interest in using our product.
We've created a plan for migrating this product from React-Bootstrap to Reactstrap a while ago, and we've allocated for it two weeks or so. We are going to start it's migration on 4th of April, and it should be done on 18th of April. The problem is that at the moment we are developing other products, and that is why the migration of this product will start on 4th of April.
So to answer your question, i think two weeks would be a fair estimate.

Best,
Manu

**Comment by ruegreen** [comment]
You note above that this will be done by April 18th, has this been done yet ?

**Comment by einazare** [comment]
Hello @ruegreen ,

We've changed our plan due to the fact that react-bootstrap is going to release a stable version for bootstrap 4.
At first we wanted to change this product from react-bootstrap (BS3) to reactstrap (BS4), but with the upcoming release of react-bootstrap (BS4) we've decided to wait and keep this product on react-bootstrap. So as soon as react-bootstrap releases their BS4 version, we are going to updated our product as well.

Best,
Manu
